minsar/utils/download_asfserial_rsmas.py

```diff
@@ -67,15 +88,64 @@ def main(iargs=None):
     return None
 
 
-def generate_files_csv(slc_dir, custom_template_file):
+def generate_files_csv(slc_dir, custom_template_file, tupleParam):
```

Comment:
Needs a more descriptive name. `tupleParam` tells me nothing about what data that is

Reply:
`def generate_files_csv(slc_dir, custom_template_file, tuple_inputs):`
This tuple contains a lot of different values in order to avoid globalization - a bit difficult to name. Kindly let me know if you come up with a better one. I chose since there are two types of inputs. standard_inputs when no seasonal dates are provided and seasonal_inputs when they are.